l10n/bundle.l10n.json

```diff
@@ -0,0 +1,85 @@
+{
+  "Cannot load Razor language server because the directory was not found: '{0}'": "Cannot load Razor language server because the directory was not found: '{0}'",
```

Comment:
@Cosifne The `keys` of this file can be used to help organize who owns the strings / help what the usage should be. 

See https://github.com/microsoft/vscode-cpptools/blob/ae904ec3306d563db44445dac90728ec706424d3/Extension/package.nls.json

Reply:
This key is auto-generated by the l10n tool, let me see if this can be optimized or not

Reply:
@Cosifne Sorry I'm incorrect here. I think this is correct for the `l10n.json` file. I got this confused with the `nls.json` file.